docs/proposals/scheduling/policy-effetive-strategy/statization-of-propagation-policy.md

```diff
@@ -0,0 +1,419 @@
+  participant Karmada
+
+  Policy1 ->> Karmada: create
+  activate Karmada
+  DeployA ->> Karmada: create
+  Karmada -->> DeployA: propagate it by Policy1
+  deactivate Karmada
+  
+  Policy1 ->> Karmada: delete
+  activate Karmada
+  loop
+    Policy1 --> Karmada: no action util DeployA updated
+  end
+  deactivate Karmada
+
+  DeployA ->> Karmada: update  
+  activate Karmada
+  Karmada -->> DeployA: add to waiting list and delete existing replicas
+  deactivate Karmada
+```
+
+## Risks and Mitigations
+
+### Compatibility risk
+
+Risk：The current version of Propagation Policy takes effect immediately and does not support preemption by default. 
+The introduction of this feature is equivalent to Propagation Policy not taking effect immediately 
+and supporting preemption by default, which is a big change in behavior and has the risk of breaking compatibility.
+
+Mitigations：
+
+* **v1.8：control this feature by a feature-gate, turn off by default, advising users to turn on.**
+* **v1.9：control this feature by a feature-gate, turn on by default, allowing users to turn off in special cases.**
+
+> The previous design was not good enough, this proposal makes more sense. Even if it destroys compatibility,
+it has to be changed to benefit more users in the future
+
+
+## Questions
+
+### How to identify whether ResourceTemplate was modified by a user or by Karmada itself
+
+The new Policy will take effect when ResourceTemplate updated, but Karmada itself can also modify the ResourceTemplate,
+e.g., adding a label, updating the status field.
+
+The desired result is that the new Policy should take effect only by users' change, not by Karmada's own change.
+
+So how can you tell if a ResourceTemplate was modified by a user or by Karmada itself?
+
+Answers：
+
+* if the label is typed by Karmada itself, it should have Karmada's prefix, e.g. karmada.io.xxxx. Thus, such updates should be ignored.
+* updates of status field should be ignored.
+* Theoretically, we should not modify the user's ResourceTemplate, if there is a need to add something to it in the future, 
+  there must be a way to clearly distinguish that this is a field modified by Karmada in the design stage. If you can't distinguish it, 
+  you should choose other ways to realize it to avoid modifying the user's ResourceTemplate.
+
+### How to identify whether ResourceTemplate is in effect under the current Policy or the previous version
+
+Answers: 
+
+* First of all, users need to understand that Policies are static and do not take effect immediately, 
+and that the distribution of user's ResourceTemplates is inert. 
+* Then, if you want to see the ResourceTemplate is in effect under which version Policy , check it at ResourceBinding.
+
+### How to put the Policy into effect immediately in special cases
+
+Answers:
+
+In the future, a new `karmadactl` command could be added to support proactively triggering the reconciliation of all ResourceTemplates managed by a given policy.
```

Comment:
I think this `reconcile` plan should be included in this proposal as it is relevant and people will need it.